charts/victoria-metrics-anomaly/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Next release
 
+- TODO
+
+## 1.6.2
+
+**Release date:** 2024-10-22
+
+![AppVersion: v1.17.2](https://img.shields.io/static/v1?label=AppVersion&message=v1.17.2&color=success&logo=)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
```

Comment:
Are these changes expected in this PR?
Just want to avoid breaking release bot if that's not expected

Reply:
rebasing fixed it